scripts/tokenomics-emulation/accounts.ts

```diff
@@ -0,0 +1,27 @@
+import { Keyring } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+
+class Account {
+  public readonly accounts = [];
```

Comment:
It should be a separate field for each account. For instance, `rootAccount`, `sudoAccount`